Makefile.am

```diff
@@ -0,0 +1,144 @@
+SUBDIRS = cppForSwig
+PREFIX = @prefix@
+
+all:
+	$(MAKE) -C cppForSwig
+
+clean-local:
+	rm -f CppBlockUtils.py
+	rm -f qrc_img_resources.py
+	rm -f _CppBlockUtils.so
+	rm -f cppForSwig/cryptopp/a.out
+	rm -f *.pyc BitTornado/*.pyc bitcoinrpc_jsonrpc/*.pyc ui/*.pyc
+	rm -f armoryengine/*.pyc dialogs/*.pyc BitTornado/BT1/*.pyc
+	rm -f pytest/*.pyc txjsonrpc/*.pyc jsonrpc/*.pyc txjsonrpc/web/*.pyc
+
+install-data-local:
+	$(MKDIR_P) $(DESTDIR)$(datarootdir)/applications
+	$(MKDIR_P) $(DESTDIR)$(datarootdir)/armory/img
+	rsync -rupE --exclude="img/.DS_Store" img $(DESTDIR)$(datarootdir)/armory/
+	$(SED) "s:python /usr:python $(PREFIX):g" < dpkgfiles/armory.desktop > $(DESTDIR)$(datarootdir)/applications/armory.desktop
```

Comment:
When I create a whatever.desktop.in and use AC_CONFIG_FILES to process an exec line that uses `@bindir@`, I end up with a line like `Exec=${exec_prefix}/bin/armory`. So I guess to fully expand the variable, I have to do something like what is outlined in [automake manual section 9.1](http://www.gnu.org/software/automake/manual/html_node/Scripts.html)?

I just made a commit that does it the way section 9.1 says.